schema/types/isolator.go

```diff
@@ -24,6 +24,7 @@ var (
 	isolatorMap map[ACIdentifier]IsolatorValueConstructor
 
 	ErrIncompatibleIsolator = errors.New("isolators set contains incompatible types")
+	ErrInvalidIsolator      = errors.New("invalid isolator")
```

Comment:
Could you add a docstring for this error? hinting that it might be thrown if an Isolator is constructed incorrectly for example